packages/default-theme/src/cms/elements/CmsElementImage.vue

```diff
@@ -1,13 +1,13 @@
 <template>
-  <div v-if="imgUrl">
+  <div v-if="imagesUrl && imagesUrl.length">
```

Comment:
it's a good practice to use optional chaining :)
```suggestion
  <div v-if="imagesUrl?.length">
```

Reply:
`<div v-if="imagesUrl?.length">` this code can not work, the terminal always said `Compiled with some errors`. I think the app does not allow us to use optional chaining in `v-if`.

Reply:
alright! So let's leave it like that! :)